docs/content/latest/api/ycql/expr_fcall.md

```diff
@@ -60,6 +61,28 @@ function_call ::= function_name '(' [ arguments ... ] ')'
 | MIN | Returns the minimum value of column values |
 | MAX | Returns the maximum value of column values |
 
+
+## Partition_hash function
+`partition_hash` is a function that takes as arguments the values of partition keys of a row and returns an `uint16` hash value.
+The full hash partition value spans from `0 - 65535`. Each tablet keeps are range of values and the hash is used to decode on which tablet the row will reside.  
```

Comment:
Each tablet keeps... (typo here).

Reword to:

Tables are partitioned into tablets, with each tablet being responsible for a range of partition values. The partition_hash of the row is used to decide which tablet the row will reside in.